Makefile

```diff
@@ -29,10 +29,10 @@ $(SEARCHDB_PATH): missions/*/transcripts/* missions/shared/glossary/* backend/in
 	rm -rf $(SEARCHDB_PATH)
 	$(PYTHON) -m backend.indexer
 
-statsporn: $(mission_dirs:%=%/images/stats/graph_0.png)
+activity_graphs: $(mission_dirs:%=%/images/activity_graphs/graph_0.png)
 
-missions/%/images/stats/graph_0.png: missions/%/transcripts/* backend/stats_porn_assets/*  backend/stats_porn.py
-	$(PYTHON) -m backend.stats_porn $*
+missions/%/images/activity_graphs/graph_0.png: missions/%/transcripts/* backend/activity_graph_components/*  backend/activity_graph_generator.py
```

Comment:
I think `missions/%/images/activity_graphs/graph_0.png` isn't an amazing URL. `activity` would be better as the path component, surely? Or make it `0.png` instead of `graph_0.png`. I'd prefer the former, slightly, because it leaves space for the existence of activity things that aren't graphs. But since we are unlikely to ever have any new features in this area that's largely moot.